applications/crossbar/src/crossbar_app.erl

```diff
@@ -23,6 +23,8 @@
 %%--------------------------------------------------------------------
 -spec start(application:start_type(), any()) -> startapp_ret().
 start(_StartType, _StartArgs) ->
+    _ = kazoo_bindings:bind(kapps_maintenance:binding(migrate), crossbar_maintenance, migrate),
```

Comment:
At least for me, in emacs, tickied atoms show up differently from raw atoms. When it was just @k-anderson and I, I liked tickied atoms for that visual cue and he didn't seem to have an opinion, so we standardized on tickied atoms.